java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/SqlServerValidator.java

```diff
@@ -315,7 +315,12 @@ private boolean isDataTypeCompatible(String ssDataType, Data.DataType.TypeName t
                 return val == Data.DataType.TypeName.DOUBLE_VALUE;
             case "decimal":
             case "numeric":
-                return val == Data.DataType.TypeName.DECIMAL_VALUE;
+                return val == Data.DataType.TypeName.DECIMAL_VALUE
```

Comment:
also for sqlserver, thanks
the existing `e2e_test/source_line` include both mysql and sqlserver